examples/show_data.py

```diff
@@ -17,13 +17,37 @@
     type=click.Path(exists=True),
     help="Path to the image to open in napari annotator",
 )
-def run_napari_annotator(image_path=str) -> None:
-    # Expects the image data & H5 node positions in the same folder.
-    # Use identical naming convention for files & specify whole path to mrc file:
-    # e.g. /Users/kulicna/Desktop/dataset/shape_squares/MRC_Synthetic_File_000.mrc
+@click.option(
+    "--show_features",
+    type=bool,
+    default=True,
+    help="Show feature representation of individual nodes",
+)
+def run_napari_annotator(
+    image_path: str | Path,
+    show_features: bool = False,
+) -> None:
+    """Function to open an image & annotate it in napari.
 
+    Parameters
+    ----------
+    image_path : str | Path
+        Absolute filename of the image to be opened.
+        Expects the image data & H5 node positions in the same folder.
+        Use identical naming convention for these files to pair them up.
+    show_features : bool
+        Whether to display node features stored in the `h5` file.
+        Defaults to False.
+
+    Notes
+    -----
+    - expected file organisation:
+        /path/to/your/image/MRC_Synthetic_File_000.mrc
+        ...identical to...
+        /path/to/your/nodes/MRC_Synthetic_File_000.h5
+    """
     # Process the image data + load nodes:
-    suffix = image_path.split(".")[-1]
+    suffix = str(image_path).split(".")[-1]
```

Comment:
I wonder why do you have to do `str(image_path)` if you have typed annotated the function input as a string already?

Reply:
In case someone wants to reuse the function from outside the command line, and inputs `Path` object as the argument (those can't be split).

Reply:
In case someone wants to reuse the function from outside the command line, and inputs `Path` object as the argument (those can't be split).

Reply:
Ohh, right I only saw the str type annotation, sorry!...